kibble/go.sum

```diff
@@ -24,6 +24,8 @@ github.com/gosimple/slug v1.1.1 h1:fRu/digW+NMwBIP+RmviTK97Ho/bEj/C9swrCspN3D4=
 github.com/gosimple/slug v1.1.1/go.mod h1:ER78kgg1Mv0NQGlXiDe57DpCyfbNywXXZ9mIorhxAf0=
 github.com/gregjones/httpcache v0.0.0-20170926212834-c1f8028e62ad h1:FxOhWCJ0DGg6vOchS5Gjtko5Yk2nBhls+oSQS4TrSsI=
 github.com/gregjones/httpcache v0.0.0-20170926212834-c1f8028e62ad/go.mod h1:FecbI9+v66THATjSRHfNgh1IVFe/9kFxbXtjV0ctIMA=
+github.com/grokify/html-strip-tags-go v0.0.1 h1:0fThFwLbW7P/kOiTBs03FsJSV9RM2M/Q/MOnCQxKMo0=
+github.com/grokify/html-strip-tags-go v0.0.1/go.mod h1:2Su6romC5/1VXOQMaWL2yb618ARB8iVo6/DR99A6d78=
```

Comment:
What are these for?

Reply:
Somebody added `grokify` to `go.mod` recently and apparently forgot to commit the `go.sum` changes.